### PR TITLE
Fix migrate channel id script

### PR DESCRIPTION
**Pull request by mparent61**
Good idea to actually test the script before commit!

### CODE REVIEW

**Comment by mparent61** [comment]
@MichaelButkovic @benghaziboy 

And one more migration fix

File: videohub_client/management/commands/migrate_channel_id.py [hunk 1]
```diff
@@ -36,16 +38,20 @@ def handle(self, *args, **options):
...
-                raise CommandError('Video request failed: {} --> {} {}'.format(
+                self.stderr.write(self.style.ERROR('Video request failed: {} --> {} {}'.format(
                     url,
                     resp.status_code,
-                    resp.reason))
+                    resp.reason)))
+                failed = True
+
+        if failed:
+            raise CommandError('Failed to update 1 or more videos')
```
Comment (MichaelButkovic) [inline videohub_client/management/commands/migrate_channel_id.py]: if you want, you could have some kind of array of failed videos and print that out, so you can check why they failed.

**Comment by MichaelButkovic** [comment]
LGTM

File: videohub_client/management/commands/migrate_channel_id.py (hunk 1 above)
Comment (mparent61) [inline videohub_client/management/commands/migrate_channel_id.py]: Yeah good idea, i'll print failed IDs, and stderr messages give more details

**Pull request merged**